presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -152,4 +153,17 @@ public double getMinimumAssignedSplitWeight()
     {
         return minimumAssignedSplitWeight;
     }
+
+    @Config("iceberg.enable-parquet-dereference-pushdown")
+    @ConfigDescription("enable parquet dereference pushdown")
+    public IcebergConfig setParquetDereferencePushdownEnabled(boolean parquetDereferencePushdownEnabled)
+    {
+        this.parquetDereferencePushdownEnabled = parquetDereferencePushdownEnabled;
+        return this;
+    }
+
+    public boolean isParquetDereferencePushdownEnabled()
+    {
+        return this.parquetDereferencePushdownEnabled;
```

Comment:
```suggestion
        return parquetDereferencePushdownEnabled;
```

Reply:
@beinan done, Thank you for your review.